x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -10,13 +10,15 @@ export const FINDINGS_ROUTE_PATH = '/api/csp/findings';
 export const BENCHMARKS_ROUTE_PATH = '/api/csp/benchmarks';
 export const UPDATE_RULES_CONFIG_ROUTE_PATH = '/api/csp/update_rules_config';
 
-export const CSP_KUBEBEAT_INDEX_PATTERN = 'logs-cis_kubernetes_benchmark.findings*';
-export const AGENT_LOGS_INDEX_PATTERN = '.logs-cis_kubernetes_benchmark.metadata*';
-export const LATEST_FINDINGS_INDEX_PATTERN = 'cloud_security_posture-findings_latest';
-export const BENCHMARK_SCORE_INDEX_PATTERN = 'cloud_security_posture-benchmark_scores';
-
 export const CSP_FINDINGS_INDEX_NAME = 'findings';
 export const CIS_KUBERNETES_PACKAGE_NAME = 'cis_kubernetes_benchmark';
+export const LATEST_FINDINGS_INDEX_NAME = 'cloud_security_posture.findings_latest';
+export const BENCHMARK_SCORE_INDEX_NAME = 'cloud_security_posture.scores';
+
+export const AGENT_LOGS_INDEX_PATTERN = '.logs-cis_kubernetes_benchmark.metadata*';
+export const CSP_KUBEBEAT_INDEX_PATTERN = 'logs-cis_kubernetes_benchmark.findings*';
```

Comment:
sorry for missing that in the first time.
Let's use the new beat name `CLOUDBEAT`

Reply:
as suggested by @CohenIdo we'll do that in a different PR